pyspark/bigdl/util/common.py

```diff
@@ -118,6 +118,13 @@ def get_dtype(bigdl_type):
     # Always return float32 for now
     return "float32"
 
+
+class JActivity(object):
```

Comment:
I don't think we need JActivity on the Python size?

Reply:
We do need that as kind of container otherwise there's no place to put those converting method.